RadixWallet/Clients/OnLedgerEntitiesClient/Helpers/OnLedgerEntitiesClient+ComplexTypes.swift

```diff
@@ -64,11 +68,13 @@ extension OnLedgerEntitiesClient {
 				guarantee: guarantee
 			)
 		}
+
 		// Normal fungible resource
 		let isXRD = resourceAddress.isXRD(on: networkID)
+		let isHidden = hiddenAssets.contains(.fungible(resourceAddress))
```

Comment:
We already need to await for getting the `hiddenAssets` in this method, since they need to be sent to the `func poolUnit()`. Then, using the helper `func isAssetHidden()` here means adding an extra await when there is no need.